src/components/vms/hostvmslist.tsx

```diff
@@ -82,6 +87,41 @@ const VmState = ({
     );
 };
 
+const VmUsageCpu = ({ vm } : {vm: VM | UIVM, }) => {
+    if (vm.isUi || vm.state != "running")
+        return null;
+
+    const vmCpuUsage: number = vm.cpuUsage ? vm.cpuUsage : 0;
+    const cpuUsage = vm.cpuUsage && isNaN(vmCpuUsage) ? 0 : parseFloat(vmCpuUsage.toFixed(1));
+
+    return (
+        <Progress value={cpuUsage}
+            aria-label={_("CPU")}
+            className="pf-m-bg ct-simple-percentage-progress"
+            min={0} max={100}
+            variant={cpuUsage > 90 ? ProgressVariant.danger : undefined}
+
+        />
+    );
+};
+
+const VmUsageMem = ({ vm } : {vm: VM | UIVM, }) => {
+    if (vm.isUi || vm.state != "running")
+        return null;
+
+    const memTotal = vm.currentMemory ? vm.currentMemory * 1024 : 0;
```

Comment:
This added line is not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-2183-42d0cd67-20250902-101715-fedora-42-devel/Coverage/lcov/github-pr.diff.gcov.html)